src/Network/Response.php

```diff
@@ -538,13 +538,18 @@ protected function _sendHeader($name, $value = null)
 
     /**
      * Sends a content string to the client.
+     * If the content is a callable, it is invoked.
      *
      * @param string $content string to send as response body
      * @return void
      */
     protected function _sendContent($content)
     {
-        echo $content;
+        if (!is_string($content) && is_callable($content)) {
```

Comment:
Right, but if you allows string callables they could conflict with an actual response, even though this is likely rare.

I feel the best way to handle this is to type check against `Closure`, and not allow all these other types of callables. It's not really necessary, and if you REALLY want to call a class or function, you can just do that form within your closure. What do you think?